src/pages/partners/affiliate-ib/_minimum-volume.js

```diff
@@ -0,0 +1,247 @@
+    padding: 0 0 8rem;
+    @media ${device.tabletL} {
+        padding: 40px 0;
+    }
+`
+const TitleWrapper = styled(Flex)`
+    max-width: 120rem;
+    margin: 0 auto 2.4rem;
+
+    ${Text} {
+        max-width: 792px;
+        margin: 0 auto;
+    }
+
+    @media ${device.tabletL} {
+        margin: 0 auto 16px;
+
+        ${Header} {
+            font-size: 20px;
+            line-height: 25px;
+        }
+    }
+`
+const StyledHeader = styled(Header)`
+    @media ${device.mobileL} {
+        margin: 0 auto 16px;
+        text-align: left !important;
+    }
+`
+const ExampleWrapper = styled(Flex)`
+    max-width: 120rem;
+    margin: 0 auto;
+    flex-direction: column;
+
+    @media ${device.laptopM} {
+        h1 {
+            text-align: center;
+        }
+    }
+`
+const CalculationWrapper = styled(Flex)`
+    @media ${device.laptopM} {
+        flex-direction: column;
+    }
+`
+const LeftWrapper = styled(Flex)`
+    width: 588px;
+    padding: 6.6rem 4rem 7.8rem;
+    background-color: var(--color-grey-39);
+    margin-right: 2.4rem;
+
+    @media (max-width: 1360px) {
+        width: 45%;
+    }
+
+    @media ${device.laptopM} {
+        width: 100%;
+        max-width: 58.8rem;
+        margin-right: 0;
+        margin-bottom: 0.8rem;
+        padding: 1.6rem;
+    }
+    @media ${device.tabletL} {
+        margin-bottom: 0;
+    }
+
+    ${Text} {
+        @media ${device.tabletL} {
+            max-width: 295px;
+        }
+    }
+`
+const RightWrapper = styled(Flex)`
+    width: 588px;
+    flex-direction: column;
+
+    @media ${device.laptopM} {
+        width: 100%;
+        max-width: 58.8rem;
+    }
+`
+const MinimumVolume = () => {
+    return (
+        <StyledSection>
+            <Container direction="column">
+                <TitleWrapper direction="column">
+                    <Header type="page-title" align="center" mb="0.8rem">
+                        {localize('Minimum volume requirement')}
+                    </Header>
+                    <Text size="16px" align="center">
+                        <Localize translate_text="To receive the minimum commission (0.01 in any currency) allowed by the system, the minimum volume requirement is calculated based on the following formulas:" />
+                    </Text>
+                </TitleWrapper>
+                <ExampleWrapper>
+                    <StyledHeader type="main-paragraph" mb="2.4rem">
+                        {localize('Example:')}
+                    </StyledHeader>
+                    <CalculationWrapper ai="center" mb="24px">
+                        <LeftWrapper>
+                            <Text size="16px">
+                                {localize(
+                                    'A deal for 1 lot of BTC/USD (with a BTC to USD exchange rate of USD 50,000) per USD 100,000 turnover will pay out a commission of USD 20. The minimum volume required to receive the minimum commission of USD 0.01 is determined using the following formula:',
```

Comment:
im afraid this might have problem during parsing later, can we have it in one line

Reply:
Changed to <Localize> comp since the {localize} still breaks due to prettier.Tried prettier-ignore but its not working.